README.md

```diff
@@ -38,3 +38,72 @@ There are two ways that you can run this application, one of them is to run all
 - `make get-data`: This function gets the initial data from [Convokit](https://convokit.cornell.edu/documentation/supreme.html)
 - `make clean-data`: This function cleans the downloaded [Convokit](https://convokit.cornell.edu/documentation/supreme.html)
 - `make describe-data`: This function parses the cleaned [Convokit](https://convokit.cornell.edu/documentation/supreme.html) data and produces an Excel file that contains basic descriptive statistics of the data.
+
+## Project Structure
```

Comment:
I don't think we need a file per file description in the `README.md`, tbh. You could do a package level description, but even that I feel may not be necessary as project structures are prone to change and the names of the folders, etc. should be self-describing (presuming we have doc strings, comments, etc., which we do).

Reply:
A lot of this description is already in the code as well, so it may end up going out of date quite quickly.

Reply:
It is also very non-standard and the descriptions for particular functions are already in the files.